selfdrive/athena/athenad.py

```diff
@@ -350,7 +365,7 @@ def uploadFileToUrl(fn, url, headers):
 
 
 @dispatcher.add_method
-def uploadFilesToUrls(files_data):
+def uploadFilesToUrls(files_data: List[Dict[str, Any]]):
```

Comment:
is Any necessary here?

Reply:
I guess `Dict[str, Union[str, bool, Dict[str, str]]]` is more exact but much more complicated

Reply:
all the more reason to do it